composer/algorithms/blurpool/blurpool_layers.py

```diff
@@ -138,7 +152,7 @@ def blurmax_pool2d(input: torch.Tensor,
                         padding=padding,
                         dilation=dilation,
                         ceil_mode=ceil_mode)
-    return blur_2d(maxs, stride=stride, filter=filter)
+    return blur_2d(maxs, channels=0, stride=stride, filter=filter)
```

Comment:
```suggestion
    return blur_2d(maxs, channels=-1, stride=stride, filter=filter)
```
super minor, but would be nice to adhere to the reshape/view convention that -1 is what indicates "infer from the input"